tensornetwork/linalg/initialization.py

```diff
@@ -0,0 +1,147 @@
+# Copyright 2019 The TensorNetwork Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Functions to initialize Tensor using a NumPy-like syntax."""
+
+import warnings
+from typing import Optional, Sequence, Tuple, Any, Union, Type, Callable, List
+from typing import Text
+import numpy as np
+from tensornetwork.backends import abstract_backend
+from tensornetwork import backend_contextmanager
+from tensornetwork import backends
+from tensornetwork.tensor import Tensor
+
+AbstractBackend = abstract_backend.AbstractBackend
+
+
+def initialize_tensor(fname: Text,
+                      *fargs: Any,
+                      backend: Optional[Union[Text, AbstractBackend]] = None,
+                      **fkwargs: Any) -> Tensor:
+  """Return a Tensor wrapping data obtained by an initialization function
+  implemented in a backend. The Tensor will have the same shape as the
+  underlying array that function generates, with all Edges dangling.
```

Comment:
tiny nit: I guess we should not use `Edge` here, could be confusing (also later)